modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/error/RosettaError.scala

```diff
@@ -42,6 +42,13 @@ object RosettaError extends IntEnum[RosettaError] with RosettaErrorEncoder {
         retriable = false,
         description = Some("Unable to decode the transaction.")
       )
+  case object ParseFailure
+      extends RosettaError(
+        value = 7,
+        message = "Parse failure",
+        retriable = false,
+        description = Some("Unable to deserialize the class to its kryo type.")
+      )
```

Comment:
What's the difference between this and MalformedTransaction?

Reply:
I will use `MalformTransaction` instead. I forgot to remove this.